lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -65,7 +65,12 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL, $fail
   // aware of a user's northstar id. If it doesn't find one, we just grab it
   // from northstar directly.
   if (!$northstar_id) {
-    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    if ($user->uid) {
+      $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    } else {
+      $northstar_user = dosomething_northstar_get_user($values['northstar_id']);
+      $user = user_load($northstar_user->drupal_id);
```

Comment:
upon even closer inspection, the code you added is actually in a chunk that is commented as a bandaid fix. maybe the check should be in the [cron job](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc#L32) instead?